



## Publications


| | | |
|--------|----------------------------------------------------------------------------------------------------------------------------|---|
|*2023*: |Kai Jing; Yewen Xu; Yang Yang; Pengfei Yin; Duo Ning; Guangyu Huang; Yuqing Deng; Gengzhan Chen; Guoliang Li; <u>**Simon Zhongyuan Tian***</un>; Meizhen Zheng*; “*ScSmOP: a universal computational pipeline for single-cell single-molecule multiomics data analysis*", ___Briefings in Bioinformatics___, (2023), 24(6) DOI: https://10.1093/bib/bbad343| [Script](https://github.com/ZhengmzLab/ScSmOP)|
|*2022*: |**Simon Zhongyuan Tian***, Pengfei Yin, Kai Jing, Yang Yang, Yewen Xu, Guangyu Huang, Duo Ning, Melissa J. Fullwood* and Meizhen Zheng*. “*MCI-frcnn: A Deep Learning Method for Topological Micro-Domain Boundary Detection*”. ___Frontiers in Cell and Developmental Biology___, (2022). DOI: https://doi.org/10.3389/fcell.2022.1050769 |[Script](https://github.com/ZhengmzLab/MCI-frcnn)|
|*2022*: |**Simon Zhongyuan Tian***, Guoliang Li, Duo Ning, Kai Jing, Yewen Xu, Yang Yang, Melissa J. Fullwood, Pengfei Yin, Guangyu Huang, Dariusz Plewczynski, Jixian Zhai, Ziwei Dai, Wei Chen* and Meizhen Zheng*. “*MCIBox: A Toolkit for Single-molecule Multi-way Chromatin Interaction Visualization and Micro-Domains Identification*”. ___Briefings In Bioinformatics___, (2022), bbac380. DOI: https://doi.org/10.1093/bib/bbac380 |[Script](https://github.com/ZhengmzLab/MCIBox)|
|*2022*: |Zhuowen Li, Yanping Long, Yiming Yu, Fei Zhang, Hong Zhang, Zhijian Liu, Jinbu Jia, Weipeng Mo, **Simon Zhongyuan Tian**, Meizhen Zheng, Jixian Zhai. “*Pore-C simultaneously captures genome-wide multi-way chromatin interaction and associated DNA methylation status in Arabidopsis*”. ___Plant Biotechnology Journal___, (2022), pbi.13811. DOI: https://doi.org/10.1111/pbi.13811||
|*2020*: |Ping Wang, Zhonghui Tang, Byoungkoo Lee, Jacqueline Jufen Zhu, Liuyang Cai, Przemyslaw Szalaj, **Simon Zhongyuan Tian**, Meizhen Zheng, Dariusz Plewczynski, Xiaoan Ruan, Edison T. Liu, Chia-Lin Wei & Yijun Ruan. “*Chromatin Topology Reorganization and Transcription Repression by PML-RARα in Acute Promyeloid Leukemia*”. ___Genome Biology___, (2020), 21, 110.  DOI: https://doi.org/10.1186/s13059-020-02030-2  ||
|*2019*: |Meizhen Zheng, **Simon Zhongyuan Tian**, Daniel Capurso, ...,Chia-Lin Wei, Yijun Ruan. “*Multiplex Chromatin Interaction Analysis with Single-Molecule Precision*”. ___Nature___, 556 (2019): 558-562. DOI: https://doi.org/10.1038/s41586-019-0949-1||
|*2019*: |Minji Kim, Meizhen Zheng, **Simon Zhongyuan Tian**, Byoungkoo Lee, Jeffrey H. Chuang & Yijun Ruan. “*MIA-Sig: multiplex chromatin interaction analysis by signal processing and statistical algorithms*”. ___Genome Biology___, (2019), 20(1): 251. DOI: https://doi.org/10.1186/s13059-019-1868-z||
|*2019*: |**Simon Zhongyuan Tian**, Daniel Capurso, Minji Kim, Byoungkoo Lee, Meizhen Zheng, Yijun Ruan. “*ChIA-DropBox: a novel analysis and visualization pipeline for multiplex chromatin interactions*”. ___BioRxiv___, (2019). DOI: https://doi.org/10.1101/613034|[Script](https://github.com/TheJacksonLaboratory/ChIA-DropBox)|
|*2017*: |Xingwang Li, Oscar Junhong Luo, Ping Wang,	Meizhen Zheng, Danjuan Wang, Emaly Piecuch, Jacqueline Jufen Zhu, **Simon Zhongyuan Tian**, Zhonghui Tang, Guoliang Li	& Yijun Ruan. “*Long-read ChIA-PET for base-pair-resolution mapping of haplotype-specific chromatin interactions*”. ___Nature Protocol___, (2017),12:899-915. DOI: https://doi.org/10.1038/nprot.2017.012 ||
|*2015*: |Tang, Zhonghui, Oscar Junhong Luo, Xingwang Li, Meizhen Zheng, …, Danjuan Wang, **Simon Zhongyuan Tian**, May Penrad-Mobayed et al. "*CTCF-mediated human 3D genome architecture reveals chromatin topology for transcription*". ___Cell___, 163, no. 7 (2015): 1611-1627. DOI: https://doi.org/10.1016/j.cell.2015.11.024 ||
|*2014*: |**Zhongyuan Tian**, and Hiroshi Matsuno. "Intelligent mechanisms in *E. coli* in processing carbon sources". ___Journal of Robotics, Networking and Artificial Life___, 1.1 (2014): 18-22. DOI: https://doi.org/10.2991/jrnal.2014.1.1.4 ||
|*2013*: |**Zhongyuan Tian**, Adrien Fauré, Hirotada Mori and Hiroshi Matsuno. "*Identification of key regulators in glycogen utilization in E. coli based on the simulations from a hybrid functional Petri net model*". ___BMC Systems Biology___, (2013), 7.6 1-10. DOI: https://doi.org/10.1186/1752-0509-7-S6-S1 ||
|*2012*: |Tomoaki Yamamotoya^#^, Hitomi Dose^#^, **Zhongyuan Tian**, et al. "*Glycogen is the primary source of glucose during the lag phase of E. coli proliferation*". ___Biochimica et Biophysica Acta (BBA)-Proteins and Proteomics___, 1824.12 (2012): 1442-1448. DOI: https://doi.org/10.1016/j.bbapap.2012.06.010||
|*2007*: |**Zhongyuan Tian**, Peijun Ma, Bin Liang. "*Map of Pattern-String Matching Algorithms*". ___Application Research of Computer___, 24.Z1, (2007)||
|*2005*: |**Zhongyuan Tian** and Bin Liang. "*Realization of VPN Multiplexing*". ___Guangdong Provincial Communications Administration 2005 Annual Report___, (2005)||
